lib/cryptoexchange/exchanges/tron_trade/services/market.rb

```diff
@@ -35,10 +35,10 @@ def adapt(output, pair)
           ticker.base      = pair.base
           ticker.target    = pair.target
           ticker.market    = Cryptoexchange::Exchanges::TronTrade::Market::NAME
-          ticker.last      = NumericHelper.to_d(output['stats']['lastPrice']) / 1_000_000
-          ticker.high      = NumericHelper.to_d(output['stats']['high']) / 1_000_000
-          ticker.low       = NumericHelper.to_d(output['stats']['low']) / 1_000_000
-          ticker.volume    = NumericHelper.to_d(output['stats']['volume']) / 1_000_000
+          ticker.last      = NumericHelper.to_d(output['stats']['lastPrice']) / 10**output['tokenDecimalsB'].to_i
+          ticker.high      = NumericHelper.to_d(output['stats']['high']) / 10**output['tokenDecimalsB'].to_i
```

Comment:
how about assigning `output['tokenDecimalsB']` to a meaningful variable name, same for A